examples/pruning/keras_integration.py

```diff
@@ -0,0 +1,94 @@
+"""
+Optuna example that demonstrates a pruner for Keras.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+Keras and MNIST, where the architecture of the neural network is optimized. Throughout the
+training of neural networks, a pruner observes intermediate results and stops unpromising trials.
+
+You can run this example as follows:
+    $ python keras_integration.py
+
+"""
+
+import optuna
+from optuna.integration import KerasPruningCallback
```

Comment:
Please move the import sentences of optuna after the import sentences of keras like [chainer example](https://github.com/pfnet/optuna/blob/master/examples/pruning/chainer_integration.py).

Reply:
Ok.